packages/expo-modules-core/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unpublished
 
+
+- Added the interface for the light sensor support on Android. ([#18225](https://github.com/expo/expo/pull/18225) by [bearkillerPT](https://github.com/bearkillerPT))
```

Comment:
Could you move this to the "New features" section too? And also, keep the blank lines the same way as you did in `expo-sensors/CHANGELOG.md` - it looks perfect there